report_aeroo/report_aeroo.py

```diff
@@ -104,14 +104,51 @@ def create_aeroo_report(
 
             filedir, filename = os.path.split(temp_file.name)
 
-            subprocess.check_output([
+            proc = subprocess.Popen([
```

Comment:
@dufresnedavid I would save the executable name in a parameter to be able to call a stub script during tests.